pkg/config/config.go

```diff
@@ -759,6 +763,10 @@ func sanitizeAPIKey(config Config) {
 	config.Set("api_key", strings.TrimSpace(config.GetString("api_key")))
 }
 
+func setLogStrippedKeys(config Config) {
```

Comment:
Why not do this in `SetupLogger` ?

Reply:
I don't know? What's the benefit?

Reply:
Having only 1 setup function instead of two

Reply:
Done.